src/labeler.ts

```diff
@@ -16,6 +16,7 @@ export async function run() {
     const token = core.getInput('repo-token');
     const configPath = core.getInput('configuration-path', {required: true});
     const syncLabels = !!core.getInput('sync-labels', {required: false});
+    const dot = !!core.getInput('dot', {required: false});
```

Comment:
We can use the `getBooleanInput` core function to parse a boolean input. Also `false` is the default value, so I think we can get rid of it.
```suggestion
    const dot = core.getBooleanInput('dot');
```

Reply:
I removed `{required: false}` in , thanks! https://github.com/actions/labeler/pull/316/commits/54d434dac50244b6843390a0d6987c951785b0f3

As of `getBooleanInput, it is currently not present in `main`. It was added there earlier, but then reverted in https://github.com/actions/labeler/pull/564. What’s the best way forward here?

Reply:
Thanks for the explanation! This change was reverted and then merged into the `v5.0.0-beta.1` branch (in scope of [this PR](https://github.com/actions/labeler/pull/574)) as the [new release](https://github.com/actions/labeler/releases/tag/v5.0.0-alpha.1) is pre-release. 

Please leave `getInput` for the `sync-labels` input because fixing it is a breaking change and it was fixed in the new major release. As for the `dot` input, please use `getBooleanInput`, because currently it does not work properly (`!!'false'` = `true`).

Reply:
Oh I see! I replaced `getInput` with `getBooleanInput` in https://github.com/actions/labeler/pull/316/commits/71d2484daaf26523ee79447d458c78a3a1b122ee, only for the `dot` option.